modules/services/waybar.nix

```diff
@@ -0,0 +1,164 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.waybar;
+
+  configText = builtins.toJSON ({ inherit (cfg) order; } // cfg.extraConfig);
+
+  configFile = pkgs.writeText "config" configText;
+
+in {
```

Comment:
It would be helpful if you add yourself as a maintainer, something like
```
meta.maintainers = [ maintainers.onny ];
```
or whatever your entry is in [maintainer-list.nix](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix).